src/OWML.Logging/ModConsole.cs

```diff
@@ -5,6 +5,7 @@ namespace OWML.Logging
 {
 	public abstract class ModConsole : IModConsole
 	{
+		[Obsolete("Use ModHelper.Console instead.")]
 		public static ModConsole OwmlConsole { get; private set; }
```

Comment:
i need this for QSB...

Reply:
Aye, removed it. Will think about it later.